src/player.cpp

```diff
@@ -3837,7 +3837,7 @@ void player::set_cat_level_rec(const std::string &sMut)
     if (!has_base_trait(sMut)) { //Skip base traits
         const auto &mdata = mutation_branch::get( sMut );
         for( auto &elem : mdata.category ) {
-            mutation_category_level[elem] += 8;
```

Comment:
I see at least one instance where it is divided by 12, adjusting that seems tricky. Why go through the trouble and change it at all? If it bothers you in the shown message, adjust it there: print `elem.second/8`. Btw. using the value 8 here allows for future mutation to add different values.